datasets/sentinel-5p/collection/template.json

```diff
@@ -0,0 +1,238 @@
+        }
+    ],
+    "stac_extensions": [
+        "https://stac-extensions.github.io/sat/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/table/v1.2.0/schema.json",
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+    ],
+    "keywords": [
+        "ESA",
+        "Copernicus",
+        "Sentinel",
+        "Air Quality",
+        "Climate Change",
+        "Forecasting"
+    ],
+    "msft:short_description": "Sentinel-5P Level 2 atmospheric monitoring products in NetCDF format",
+    "msft:storage_account": "sentinel5euwest",
+    "msft:container": "sentinel-5p",
+    "msft:region": "westeurope",
+    "providers": [
+        {
+            "name": "ESA",
+            "roles": [
+                "producer",
+                "processor",
+                "licensor"
+            ],
+            "url": "https://earth.esa.int/web/guest/home"
+        },
+        {
+            "name": "Microsoft",
+            "roles": [
+                "host"
+            ],
+            "url": "https://planetarycomputer.microsoft.com"
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Sentinel-5P Level-2 NetCDF Thumbnail",
+            "href": "https://ai4edatasetspublicassets.blob.core.windows.net/assets/pc_thumbnails/sentinel-5p-l2-netcdf-thumb.png",
+            "media_type": "image/png"
+        },
+        "geoparquet-items": {
+            "href": "abfs://items/sentinel-5p-l2-netcdf.parquet",
+            "title": "GeoParquet STAC Items",
+            "description": "Snapshot of the collection's STAC items exported to GeoParquet format.",
+            "type": "application/x-parquet",
+            "roles": [
+                "stac-items"
+            ],
+            "table:storage_options": {
+                "account_name": "pcstacitems"
+            },
+            "msft:partition_info": {
+                "is_partitioned": true,
+                "partition_frequency": "MS"
+            }
+        }
+    },
+    "summaries": {
+        "constellation": [
+            "Sentinel-5P"
+        ],
+        "platform": [
+            "Sentinel 5 Precursor"
+        ],
+        "instruments": [
+            "TROPOMI"
+        ],
+        "sat:platform_international_designator": [
+            "2017-064A"
+        ],
+        "s5p:collection_identifier": [
+            "01",
+            "02",
+            "03"
+        ],
+        "s5p:processing_mode": [
+            "NRTI",
+            "OFFL",
+            "RPRO"
+        ],
+        "s5p:product_type": [
+            "L2__AER_AI",
+            "L2__AER_LH",
+            "L2__CH4___",
+            "L2__CLOUD_",
+            "L2__CO____",
+            "L2__HCHO__",
+            "L2__NO2___",
+            "L2__NP_BD3",
+            "L2__NP_BD6",
+            "L2__NP_BD7",
+            "L2__O3_TCL",
+            "L2__O3____",
+            "L2__SO2___"
+        ],
+        "s5p:spatial_resolution": [
+            "5.5x3.5km",
```

Comment:
I wonder about the `<horizontal>x<vertical><unit>` pattern here. Is there anything else like this in STAC? Is it better to break that into something more structured?

If we keep this, thoughts on a space vs. no space around the `x` and before the `km`?

Reply:
>Is there anything else like this in STAC?

Not directly. There is a `spatial_resolution` field in `raster:bands` (defined as _"Average spatial resolution (in meters) of the pixels in the band."_), but as discussed elsewhere, `raster:bands` is ambiguous as to which band in the NetCDF the information is referencing. 

There is also a `resolution` field in sentinel-3: 
```
      "resolution": [
        1000,
        1000
      ],
```

which is in meters. It's not clear which is which (horizontal vs vertical). But it would be nice to decide on a common solution for sentinel-5 and sentinel-3. I think the list of numbers is nicer than the text string. And specifying distances in meters (not kilometers) is pretty standard in STAC. So maybe a custom spatial resolution field that could go in properties or on an asset?

- `"s5p:spatial_resolution": [5500, 3500]`
- `"s3:spatial_resolution": [1000, 1000]`

Reply:
Done.